charts/portal/README.md

```diff
@@ -461,6 +461,9 @@ this feature.
 | `rabbitmq.serviceAccount.create`| Enable creation of ServiceAccount for Bitnami RabbitMQ |`true`|
 | `rabbitmq.serviceAccount.name`| Name of the created serviceAccount | Generated using the `portal.fullname` template |
 | `rabbitmq.rbac.create`| Create & use RBAC resources |`true`|
+| `rabbitmq.volumePermissions.enabled` | Enable init container that changes the owner and group of the persistent volume(s) mountpoint to `runAsUser:fsGroup`. After enabling set the appropriate runAsUser and fsGroup values |`false`|
```

Comment:
Can we document persistent parameters added in [12.0.3](https://github.com/bitnami/charts/tree/2858f965ea5503bd91006fd01de59562260c4cb6/bitnami/rabbitmq) which were not there in [7.6.6](https://github.com/bitnami/charts/tree/1b9ca8daea338aed3949b1857a94c8a5ac451fdc/bitnami/rabbitmq)?

Reply:
@as666499 Only 'mountPath' is the difference. Added that part now.